Scatt.java

```diff
@@ -27,13 +27,13 @@
      */
     public static void main(String[] args)
     {
-        System.out.println("Please enter the folder name: ");
+        System.out.print("Please enter the folder path: ");
         String dirName = System.console().readLine();
```

Comment:
You are totally right - I completely forgot about this. We not implement this change and include the note about the CWD in the instructions?